client/src/components/FlexTableIconCell.js

```diff
@@ -8,11 +8,11 @@ const FlexTableIconCell = ({ title, item, iconClick }) => {
       <div className={`flexTable-cell--content ${title.fieldName}-content`}>
         {title.icons?.map((icon, i) => (
           <Icon
-            key={i}
-            name={item.id}
+            title={icon}
+            key={i}           
```

Comment:
@salih18 lets still go with Frank's suggestion here to use `item.id` as the `key` prop - it is safer than using the index (https://reactjs.org/docs/lists-and-keys.html#keys)

Reply:
Sorry I forgot to make an explanation. The icons are rendered on the same page and if i put id as a key, the same keys will be on the same page. so that I cannot do that.